src/modules/number/index.ts

```diff
@@ -375,14 +375,18 @@ export class NumberModule extends SimpleModuleBase {
    * @param options Maximum value or options object.
    * @param options.min Lower bound for generated bigint. Defaults to `0n`.
    * @param options.max Upper bound for generated bigint. Defaults to `min + 999999999999999n`.
+   * @param options.multipleOf The generated bigint will be a multiple of this parameter. Defaults to `1n`.
    *
    * @throws When `min` is greater than `max`.
+   * @throws When `multipleOf` is not a positive bigint.
```

Comment:
The multiple of parameter can technically be defined as string...

```suggestion
   * @throws When `multipleOf` is not positive.
```